Makefile

```diff
@@ -180,6 +180,10 @@ distclean: clean ## fully cleans up project tree
 .PHONY: distclean
 endif
 
+bundle: image ## build the changelog
+	$(run) sh -c 'hab pkg install core/aws core/coreutils -b \
+		&& /src/support/create_bootstrap_bundle.sh $(VERSION)
```

Comment:
I think you could drop the `/src` part and leave it as relative. Should make it workable inside or outside a Studio.